xonsh/xontribs.py

```diff
@@ -1,13 +1,13 @@
 """Tools for helping manage xontributions."""
-import os
 import sys
 import json
 import builtins
 import argparse
 import functools
 import importlib
 import importlib.util
-
+import xonsh.xontribs_meta
```

Comment:
Because `xontribs_meta` is at the same level as this module (xontribs) you should import the functions that you need directly, as below with tools. So this should be `from xonsh.xontribs_meta import f1, f2, ...`